CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
         ([#973](https://github.com/Automattic/pocket-casts-android/pull/973)).
     *   Improved the Automotive user profile view
         ([#975](https://github.com/Automattic/pocket-casts-android/pull/975)).
+    *   Added audio ducking as an option when playing over external notifications
+        ([#1009](https://github.com/Automattic/pocket-casts-android/pull/1009)).
```

Comment:
We'll need to merge in `main` and move this entry to the next release before we merge this.